leakcanary-sample/src/androidTest/java/com/example/leakcanary/tests/TuPeuxPasTest.kt

```diff
@@ -37,7 +36,7 @@ import androidx.test.espresso.matcher.ViewMatchers.withText
  */
 class TuPeuxPasTest {
 
-  @Rule
+  @get:Rule
```

Comment:
We should be able to use just `@Rule` if we add the `@JvmStatic` annotation

Reply:
JUnit probably wouldn't like a static rule field, and anyway I'd rather avoid static fields in tests :)